lib/StripeResource.js

```diff
@@ -212,6 +212,20 @@ StripeResource.prototype = {
     }
   },
 
+  _errorHandlerStream: function(callback) {
```

Comment:
Per comment further down, I think the `StreamProcessingError` should be created (and wrap the incoming error) in `FileUploads.js`, since it's so specific to FileUploads.

Reply:
(Apparently I meant 'further up' :joy:)

Reply:
good point! - removed and updated `StreamProcessingError` to FileUploads.